pipeline.py

```diff
@@ -209,6 +209,8 @@ def ilab_pipeline(
     )
     sdg_task.set_env_variable("HOME", "/tmp")
     sdg_task.set_env_variable("HF_HOME", "/tmp")
+    add_toleration_json(sdg_task, train_tolerations)
+    add_node_selector_json(sdg_task, train_node_selectors)
```

Comment:
The SDG task should request the accelerator too
```suggestion
    sdg_task.set_accelerator_type(eval_gpu_identifier)
    sdg_task.set_accelerator_limit(1)
    add_toleration_json(sdg_task, train_tolerations)
    add_node_selector_json(sdg_task, train_node_selectors)
```